models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -135,20 +135,18 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
-                                                                              0.001) / timestep.s,  # Autotrophic Respiration in kgC/m2/s
-      "HeteroResp" = ((sub.sipnet.output$rSoil - sub.sipnet.output$rRoot) * 0.001) / timestep.s,  # Heterotrophic Respiration in kgC/m2/s
-      "SoilResp" = (sub.sipnet.output$rSoil * 0.001) / timestep.s,  # Soil Respiration in kgC/m2/s
-      "NEE" = (sub.sipnet.output$nee * 0.001) / timestep.s,  # NEE in kgC/m2/s
-      "AbvGrndWood" = (sub.sipnet.output$plantWoodC * 0.001),  # Above ground wood kgC/m2
-      "leaf_carbon_content" = (sub.sipnet.output$plantLeafC * 0.001),  # Leaf C kgC/m2
-      "TotLivBiom" = (sub.sipnet.output$plantWoodC * 0.001) + (sub.sipnet.output$plantLeafC * 0.001) + 
-        (sub.sipnet.output$coarseRootC + sub.sipnet.output$fineRootC) * 0.001, # Total living C kgC/m2
-      "TotSoilCarb" = (sub.sipnet.output$soil * 0.001) + (sub.sipnet.output$litter * 0.001)  # Total soil C kgC/m2
+      "GPP" = PEcAn.utils::ud_convert(sub.sipnet.output$gpp, "gC/m2", "kgC/m2") / timestep.s,  # GPP in kgC/m2/s
+      "NPP" = PEcAn.utils::ud_convert(sub.sipnet.output$gpp, "gC/m2", "kgC/m2") / timestep.s - (PEcAn.utils::ud_convert(sub.sipnet.output$rAboveground, "gC/m2", "kgC/m2") / timestep.s + PEcAn.utils::ud_convert(sub.sipnet.output$rRoot, "gC/m2", "kgC/m2") / timestep.s), # NPP in kgC/m2/s. Post SIPNET calculation
+      "TotalResp" = PEcAn.utils::ud_convert(sub.sipnet.output$rtot, "gC/m2", "kgC/m2") / timestep.s,  # Total Respiration in kgC/m2/s
+      "AutoResp" = PEcAn.utils::ud_convert(sub.sipnet.output$rAboveground, "gC/m2", "kgC/m2") / timestep.s + PEcAn.utils::ud_convert(sub.sipnet.output$rRoot, "gC/m2", "kgC/m2") / timestep.s,  # Autotrophic Respiration in kgC/m2/s
+      "HeteroResp" = PEcAn.utils::ud_convert((sub.sipnet.output$rSoil - sub.sipnet.output$rRoot), "gC/m2", "kgC/m2") / timestep.s,  # Heterotrophic Respiration in kgC/m2/s
+      "SoilResp" = PEcAn.utils::ud_convert(sub.sipnet.output$rSoil, "gC/m2", "kgC/m2") / timestep.s,  # Soil Respiration in kgC/m2/s
+      "NEE" = PEcAn.utils::ud_convert(sub.sipnet.output$nee, "gC/m2", "kgC/m2") / timestep.s,  # NEE in kgC/m2/s
+      "AbvGrndWood" = PEcAn.utils::ud_convert(sub.sipnet.output$plantWoodC, "gC/m2", "kgC/m2"),  # Above ground wood kgC/m2
+      "leaf_carbon_content" = PEcAn.utils::ud_convert(sub.sipnet.output$plantLeafC, "gC/m2", "kgC/m2"),  # Leaf C kgC/m2
+      "TotLivBiom" = PEcAn.utils::ud_convert(sub.sipnet.output$plantWoodC, "gC/m2", "kgC/m2") + PEcAn.utils::ud_convert(sub.sipnet.output$plantLeafC, "gC/m2", "kgC/m2") + 
+        PEcAn.utils::ud_convert(sub.sipnet.output$coarseRootC + sub.sipnet.output$fineRootC, "gC/m2", "kgC/m2"), # Total living C kgC/m2
+      "TotSoilCarb" = PEcAn.utils::ud_convert(sub.sipnet.output$soil, "gC/m2", "kgC/m2") + PEcAn.utils::ud_convert(sub.sipnet.output$litter, "gC/m2", "kgC/m2")  # Total soil C kgC/m2
```

Comment:
1. Same comment as above about "C" not being recognized in units.
2. These lines were quite long already and with this change they've tipped over to being ~impossible for me to read. I suggest both wrapping lines and deleting the portions of existing comments that are now explicit in the ud_convert call.